vega_sim/scenario/adhoc.py

```diff
@@ -30,17 +37,29 @@ def main():
 
     scenario.pause_every_n_steps = args.pause_every_n_steps
 
-    with VegaServiceNull(
-        run_with_console=args.console,
-        launch_graphql=args.graphql,
-        warn_on_raw_data_access=False,
-        seconds_per_block=scenario.block_length_seconds,
-        transactions_per_block=100,
-        retain_log_files=True,
-        use_full_vega_wallet=False,
-        store_transactions=args.store,
-    ) as vega:
-        scenario.run_iteration(vega=vega, pause_at_completion=args.pause)
+    if args.network == "nullchain":
```

Comment:
Assuming the list of network options is reasonably fixed I think we'd be best off making this an enum somewhere rather than passing a string around